data/ships.txt

```diff
@@ -1923,7 +1923,7 @@ ship "Modified Argosy"
 	plural "Modified Argosies"
 	sprite "ship/modified argosy"
 	attributes
-		category "Light Warship"
+		category "Medium Warship"
```

Comment:
Why are you changing a ship to an existing category, if it was wanted in this category then it would have been put in this category in the first place.

Reply:
I would agree that the Mod. Argosy is hardly a Medium Warship.

Reply:
Notably, modified argosy is on par with a gunboat or a raven in terms of health, but it carries the armament of a firebird or a corvette.

Reply:
Because I believe the modified argosy was only assigned the "light warship" category because the normal argosy was in the "light freighter" category because there used to be no "medium freighter" category.

Comparing the modified argosy with other human space warships, it's actually heavier than several medium warships, therefore I believe this change is justified.

Reply:
Mass *is not* the only thing that determines a ship's category. Otherwise, the Summer Leaf would be an interceptor, or Met Par Tek 53 and Model 32 would be medium warships, or some other ridiculous nonsense would occur. A Marauder Bounder or a Marauder Fury or even a stock Hawk would become a light warship.

Reply:
If not mass, then what? If outfits, weapons, engines, number of guns and turrets, the Modified Argosy clearly fits within the medium warship category, see below.

Reply:
In case you hadn't noticed, there is not set metric on how to define a ship.

Reply:
No rules set in stone, perhaps, but there appears to be a pattern, at least for human space ships. I suggest we follow the warship category pattern and apply this to the freighter categories.